samples/ControlCatalog/App.xaml.cs

```diff
@@ -69,10 +66,6 @@ public class App : Application
                 Source = new Uri("avares://Avalonia.Themes.Fluent/Accents/BaseDark.xaml")
             },
             new StyleInclude(new Uri("resm:Styles?assembly=ControlCatalog"))
-            {
-                Source = new Uri("avares://Avalonia.Themes.Default/Accents/BaseDark.xaml")
```

Comment:
But these resources aren't included with any other StyleInclude.
There are different styles from Fluent theme, that were included with Default on Control Catalog, so we can use Fluent-only resources on some pages.

Reply:
ok, I close the pr